simapp/simd/cmd/root.go

```diff
@@ -93,7 +93,12 @@ func NewRootCmd() *cobra.Command {
 
 	initRootCmd(rootCmd, encodingConfig.TxConfig, encodingConfig.InterfaceRegistry, encodingConfig.Codec, tempApp.BasicModuleManager)
 
-	if err := tempApp.AutoCliOpts().EnhanceRootCommand(rootCmd); err != nil {
+	// add keyring to autocli opts
+	autoCliOpts := tempApp.AutoCliOpts()
+	initClientCtx, _ = config.ReadFromClientConfig(initClientCtx)
+	autoCliOpts.Keyring = initClientCtx.Keyring
```

Comment:
upgrading.md entry?

Reply:
it also looks like this can be abstracted away and made into a single line here

Reply:
There is an error, so it cannot be abstracted away.
Agree for the UPGRADING.md, but I think it should be added here as well: https://docs.cosmos.network/main/core/cli